.gitignore

```diff
@@ -8,4 +8,5 @@ pheanstalk.phar
 vendor/
 # Test coverage is stored here
 tests/coverage
-.idea
\ No newline at end of file
+.idea
```

Comment:
Ignoring these editor-sourced directories, and files, is better done on a local user basis. You can setup a `.gitignore` file on your machine, that will also be used in every repository.
The Github help site has an example for [Configuring ignored files for all repositories on your computer](https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

I've got a sample global .gitignore file in [one of my own repos](https://github.com/alister/dotfiles/blob/master/gitignore) It is renamed to ~/.gitignore, and setup for use in my ~/.gitconfig file:
```ini
[core]
    excludesfile = /home/username/.gitignore
```

Reply:
Thanks, I'm not sure I fully agree. Having this here is not big downside, there are only a few common editors and IDEs. (more devil's advocate than a really strong opinion).

Jetbrains themselves go even further and recommend committing part of the folder so   IDE settings like line endings or tabs vs spaces are automatically shared. There is an argument to be made that it provides value to users of that IDE without costing users of other IDEs anything. Of course the other extreme would be to have lots Of IDE specific files or ignores that are essentially unrelated to this project...

I'll give it some thought.